storage/pushsync/pushsync_simulation_test.go

```diff
@@ -59,14 +60,10 @@ func TestPushSyncSimulation(t *testing.T) {
 	nodeCnt := 4
 	chunkCnt := 500
 	trials := 10
```

Comment:
I find this confusing. `trial` means that you try one and the same thing multiple times... a synonym to `attempt`.

However what this test is doing is running 10 independent uploads and downloads, and makes sure that none of them fails. So I suggest we rename this to `n` or to `testcases`, but not `trials`.

Reply:
Also in the context of Swarm, when I reads `trial` I am thinking "we are probably uploading and trying to download immediately, this is why we need to retry multiple times, as chunks might not be synced" - which is not at all what is happening here.